content/en/docs/release_notes/austen.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Austen release notes
+linkTitle: Austen
+description: 9 September 2022
+date: 2022-09-09
+Hide_readingtime: true
+---
+## Summary
+
+{{% variables/apibuilder_prod_name %}} v5 is released. Projects created on v4 are compatible with v5, but require installing upgrades for components (see [Upgrades](/docs/developer_guide/console#updates-tab)).
```

Comment:
**issue**: we want to link to the new Updates section in this summary and in the notes.